Add workaround in Prerequisites (#1668)
theforeman/foreman-documentation: guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

ifdef::satellite[]
* Review Known Issues before you begin an upgrade.
For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].

> * Leapp utility does not properly enable the {Project} module.

Reviewer: ```suggestion
* Leapp utility does not properly enable the {Project} module for {ProjectServer}.
```

Run the following command to work around the issue:

> +

Reviewer 2: ```suggestion
+
[options="nowrap", subs="attributes"]
```

> To work around to enable the {Project} module for {ProjectServer}, run the following command:

Reviewer: ```suggestion
Run the following command to work around the issue.
```

Reviewer 2: As a little detail, don't forget colon at the end.

----
# subscription-manager repo-override \
--{RepoRHEL8ServerSatelliteServerProductVersion} \
--add=module_hotfixes:1
----

> For {SmartProxyServer}:

Reviewer 2: This sentence reads incomplete. Is this a step that is supposed to be done instead of the previous step if we are upgrading smartproxy server instead of a project server?

> +

Reviewer 2: ```suggestion
+
[options="nowrap", subs="attributes"]
```

> To work around to enable the {Project} module for {SmartProxyServer}, run the following command:

Reviewer: ```suggestion
For {SmartProxyServer}:
```
Making it uniform in case the idea is approved.

ifdef::satellite[]
* Review Known Issues before you begin an upgrade.
For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
* Leapp utility does not properly enable the {Project} module for {ProjectServer}.

> Run the following command to work around the issue for {ProjectServer}:

Reviewer 2: ```suggestion
Run the following command to work around the issue on {ProjectServer}:
```

+
[options="nowrap", subs="attributes"]
----
# subscription-manager repo-override \
--{RepoRHEL8ServerSatelliteServerProductVersion} \
--add=module_hotfixes:1
----

> Run the following command to work around the issue for {SmartProxyServer}:

Reviewer 2: ```suggestion
Run the following command to work around the issue on {SmartProxyServer}:
```
